ietf/doc/views_doc.py

```diff
@@ -1153,6 +1126,9 @@ def document_email(request,name):
 
 
 def get_diff_revisions(request, name, doc):
+    """ returns list of (name, rev, time, url, is_this_doc, is_previous_doc)
```

Comment:
Would be nice to follow the [PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings) convention for docstrings as much as we can